.github/workflows/build.yml

```diff
@@ -24,7 +24,6 @@ on:
 jobs:
   build:
     name: Build Wave
-    if: "github.event == 'push' || github.repository != github.event.pull_request.head.repo.full_name"
```

Comment:
Why are we removing this line in this PR?

Reply:
Come from merging this other PR I did: https://github.com/seqeralabs/wave/pull/370

Basically, when working on a PR in this very same repo (like this one), none of the conditions is true, and hence the CI is skipped. 

(note: my workflow is commits in my local computer, followed by push to this repo, with PR open)

Reply:
I admit I am getting some head aches in completely understanding why CI workflows in our repos sometimes run and sometimes don't ... happy to take any advice! :-)

Reply:
please review https://github.com/seqeralabs/wave/pull/370 first :-)